src/software/telemetry/src/lib.rs

```diff
@@ -23,16 +24,19 @@ use std::sync::mpsc::Sender;
 use parsers::*;
 use structures::*;
 
+pub type TelemetryChannelType = Result<TelemetryMessage, serial::core::Error>;
```

Comment:
I believe `telemetry/src/bin.rs` would also need some changes to compile, right?

You can test it with `./scripts/run.sh -h`